rego-filters/Policy-Image-Name.rego

```diff
@@ -0,0 +1,13 @@
+package postee.aa
```

Comment:
should be named `Allow-Image-Name`

Reply:
also why is such `package`? - it breaks correct work of rule.
And as `postee` package is required then it should be documented somewhere

Reply:
name changed
package info added in readme